definitions/divorce/json/CaseEvent/CaseEvent-general-email-nonprod.json

```diff
@@ -0,0 +1,13 @@
+[
+  {
+    "LiveFrom": "27/08/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "createGeneralEmail",
+    "Name": "Create general email",
+    "Description": "Create general email",
+    "DisplayOrder": 15,
+    "PreConditionState(s)": "*",
+    "PostConditionState": "*",
+    "SecurityClassification": "Public"
```

Comment:
Are you sure you're not missing a call to an endpoint to trigger the emails?

Reply:
no, I can create it as another PR. Endpoint doesn't exist. If I added it here it would be blocked until DIV-6502 is finished and deployed on aat.

We need just 2 lines:

```
    "CallBackURLSubmittedEvent": "${CCD_DEF_COS_URL}/create-general-email",
    "RetriesTimeoutURLSubmittedEvent": "120,120",
```

Reply:
Also - there is no AC for callback here in the scope of this story.